### ISSUE TITLE
wordpress login

### ISSUE THREAD

**Issue by wetubeclub** [WWBN/AVideo#6779]
Hello Daniel,
Question:
the link does not work
from where I will get the credentials?

Thank you
Ilias
![word](https://user-images.githubusercontent.com/60543030/175476566-0e9e3819-64e4-49d1-9785-349108c24482.jpg)

**Comment by DanielnetoDotCom** [comment]
On this new plugin, there are 2 different types of login

1. is the global WordPress site
2. login using your custom WP site API 

for item 2 make sure you enter your My Custom WP Site

![image](https://user-images.githubusercontent.com/18009293/175541213-39a64919-c278-49ae-8753-2ea1d991e528.png)

**Comment by wetubeclub** [comment]
Hello Daniel
please guide me regarding the configuration of the plugin
when I turn it on it displays the login fields twice

Thank you
Ilias

![wordpress login](https://user-images.githubusercontent.com/60543030/175778990-0d34a1de-b5ef-47b1-87eb-0d657ee7e6bf.jpg)

![wordpress login1](https://user-images.githubusercontent.com/60543030/175779010-e7ad009b-de87-4f5d-a0a8-15255bff4868.jpg)

**Comment by shanesipe** [comment]
Where do we get this plugin I have been looking for it?

**Comment by wetubeclub** [comment]
last git pull and you will see a plugin
login wordpress
![login](https://user-images.githubusercontent.com/60543030/175781626-dee2376b-7dd4-446c-a4f3-94a8200d4704.jpg)

**Comment by DanielnetoDotCom** [comment]
Hi, 

the question is.

do you want to have 2 types of logins/signup? or only WordPress from now?

if you want to have 2, I need to improve the plugin so I will make the 2 different login forms different. right now I understand they look the same.

if you want to have only one user database (the WordPress) you need to disable the native signup on the customize user.

**Comment by wetubeclub** [comment]
Maybe I did not understand well how this plugin works
what I want is if it happens naturally
one who has a username and password from the wordpress site can log in with the same credentials in the Avideo script
that is, as it is with the forum phpbb with the same codes linked to both sites

Thank you for your response!!
Ilias

**Comment by DanielnetoDotCom** [comment]
no, they are not the same. on the PHPBB, we use the AVideo user and use it to create/authenticate a user on PHPBB (we do create a session on PHPBB)

on WP we only use the user database. we do not create or create a session on WordPress

**Comment by wetubeclub** [comment]
I got it wrong!! 
thank you Daniel

**Comment by shanesipe** [comment]
Having issues with using Wordpress plugin I have oauth plugin put key and client is and it still is not working

**Comment by shanesipe** [comment]
[30-Jun-2022 13:23:44 America/Chicago] AVideoLog::ERROR: {"type":1,"message":"Uncaught TypeError: Cannot access offset of type string on string in \/var\/www\/html\/AVideo\/objects\/login.json.php:75\nStack trace:\n#0 {main}\n  thrown","file":"\/var\/www\/html\/AVideo\/objects\/login.json.php","line":75} SCRIPT_NAME: /objects/login.json.php thats what I get when using custom wp page

**Comment by DanielnetoDotCom** [comment]
can you please share more from your log?

also make sure you update all your files with `git pull`

**Comment by shanesipe** [comment]
************ ******** **************** ***************** ***** * * *** **** * **** *** ********* ** ******** ************ *************** - pass
 ************ ******** **************** ***************** ***** * * *** **** * **** *** ********* ** ******** ************ *************** - pass
 ************ ******** **************** ***************** ***** * * *** **** * **** *** ********* ** ******** ************ *************** - pass
 [30-Jun-2022 13:25:14 America/Chicago] AVideoLog::DEBUG: Warning: Slow process detected [modeYoutubeBundle.php] On  Line 90 takes 0.2274 seconds to complete, Limit (0.05). path/to/my/streamer/site/view/index.php SCRIPT_NAME: /view/index.php
 [30-Jun-2022 13:25:14 America/Chicago] AVideoLog::DEBUG: Live::getStatsObject[0] 4: cache not found SCRIPT_NAME: /view/index.php
 [30-Jun-2022 13:25:14 America/Chicago] AVideoLog::DEBUG: Live::getStats empty obj->useLiveServers} SCRIPT_NAME: /view/index.php
 [30-Jun-2022 18:25:14 UTC] AVideoLog::DEBUG: getStart language: got from config  SCRIPT_NAME: /view/index.php
 [30-Jun-2022 18:25:14 UTC] AVideoLog::DEBUG: getStart language: got from config  SCRIPT_NAME: /view/robots.txt.php
 [30-Jun-2022 13:24:26 America/Chicago] AVideoLog::DEBUG: Start Login Request redirectUri=https://www.mysite.com/ SCRIPT_NAME: /objects/login.json.php

**Comment by shanesipe** [comment]
array(4) { ["type"]=> int(1) ["message"]=> string(147) "Uncaught TypeError: Cannot access offset of type string on string in /var/www/html/AVideo/objects/login.json.php:75 Stack trace: #0 {main} thrown" ["file"]=> string(43) "/var/www/html/AVideo/objects/login.json.php" ["line"]=> int(75) } {"error":true,"msg":"Under Maintanance"}
this is what it shows on website when I try to login

**Comment by shanesipe** [comment]
git pull

**Comment by shanesipe** [comment]
Git pull is done

**Comment by DanielnetoDotCom** [comment]
if you do not share a more complete log, I will not be able to help you

**Comment by shanesipe** [comment]
That's all it's showing if there was more I would share it.
